etc/.aliases

```diff
@@ -8,7 +8,7 @@ alias .....="cd ../../../.."
 alias python=python3
 alias pip=pip3
 alias virtualenv="virtualenv --python=`which python3`"
-alias vjahia2wp="source /srv/${WP_ENV}/venv/bin/activate && export PYTHONPATH=/srv/${WP_ENV}/jahia2wp/src && cd /srv/${WP_ENV}/jahia2wp/src"
+alias vjahia2wp="make -C /srv/${WP_ENV}/jahia2wp bootstrap-mgmt && source /srv/${WP_ENV}/venv/bin/activate && export PYTHONPATH=/srv/${WP_ENV}/jahia2wp/src && cd /srv/${WP_ENV}/jahia2wp/src"
```

Comment:
the downside here is the "waiting time" when we connect into the container and make use of the alias... waiting that pip checks the dependencies.

I am not saying it is bad, and it actually will help every time someone adds a new dep... but it still requires us to wait (don't like waiting)

@GregLeBarbar , @lboatto : what do you think ?

Reply:
Got it, I reworked the logic so that "make bootstrap-mgmt" does the PIP thing each and every time, but "make /srv/${WP_ENV}/venv" does nothing if the directory exists. The overhead is now down to one gratuitous invocation of "make", and we could guard against even that with a test -d in .alias. Kindly review again and let me know if you want the test -d.

Reply:
On my Mac:

```
www-data@4d483e0e8ef6:/srv/test$ time vjahia2wp
make: Entering directory '/srv/test/jahia2wp'
make: '/srv/test/venv' is up to date.
make: Leaving directory '/srv/test/jahia2wp'

real	0m0.027s
user	0m0.000s
sys	0m0.000s
(venv) www-data@4d483e0e8ef6:/srv/test/jahia2wp/src$ alias vjahia2wp="([ -d /srv/${WP_ENV}/venv ] || make -C /srv/${WP_ENV}/jahia2wp bootstrap-mgmt) && source /srv/${WP_ENV}/venv/bin/activate && export PYTHONPATH=/srv/${WP_ENV}/jahia2wp/src && cd /srv/${WP_ENV}/jahia2wp/src"
(venv) www-data@4d483e0e8ef6:/srv/test/jahia2wp/src$ time vjahia2wp

real	0m0.023s
user	0m0.000s
sys	0m0.000s
```

So the difference is negligible, although I guess the second form is better in that it eliminates some log noise. So I went ahead and updated the alias too.